app/frontend/wallet/byron/cardano-wallet-secret-crypto-provider.ts

```diff
@@ -101,7 +102,7 @@ const CardanoWalletSecretCryptoProvider = ({
   async function signTxGetStructured(txAux, addressToAbsPathMapper) {
     const txHash = txAux.getId()
     const witnesses = await Promise.all(
-      txAux.inputs.map(async (input) => {
+      txAux.inputs.map((input) => {
```

Comment:
I believe the "async" keyword was there to highlight that the function is returning a promise. Since this was refactored to typescript, I would add explicit type annotation to the function, probably `Promise<TxWitness>`